backends/backend.conf

```diff
@@ -4,6 +4,104 @@ backend {
   default = "Local"
   providers {
 
+    slurm_singularity {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+      config {
+        script-epilogue = "sleep 30"
+        concurrent-job-limit = 50
+        runtime-attributes = """
+          Int cpu = 1
+          Int? time
+          Int? memory_mb
+          String? slurm_partition
+          String? slurm_account
+          String singularity_container
+          String? singularity_options
+          String singularity_command = "exec"
+          String? singularity_command_options
+        """
+        submit = """
+          sbatch \
+          --export=ALL \
+          -J ${job_name} \
+          -D ${cwd} \
+          -o ${out} \
+          -e ${err} \
+          ${"-t " + time*60} \
+          -n 1 \
+          --ntasks-per-node=1 \
+          ${"--cpus-per-task=" + cpu} \
+          ${"--mem=" + memory_mb} \
+          ${"-p " + slurm_partition} \
+          ${"--account " + slurm_account} \
```

Comment:
Should partition either default to "normal" or if not specific, just not be specific in the command (and will use default determined by cluster?)

Reply:
The whole `${" " + VAR_NAME}` is ignored if `VAR_NAME` is declared optional but it's not defined. so will use default determined by cluster. For example, on SCG users need to use an account instead of a partition. On Sherlock users need to use a partition instead of an account. This backend file is designed to cover both.

Reply:
ohh super cool! Okay, understood, this is exactly the functionality we want.